sale_exception/readme/USAGE.rst

```diff
@@ -0,0 +1,23 @@
+Product warning:
+#. Go to *Sales > Products > Products*.
+#. Edit or create new product.
+#. Go to *Sales* tab.
+#. Set warning option according to *Warning when Selling this Product* group.
+#. Set some warning message.
+#. Go to *Sales > Orders > Quotations*
+#. Create new quotation.
+#. Add product with warning message.
+#. Confirm order.
+#. An exception will be displayed.
```

Comment:
From my testing, the exception is displayed when saving the quotation, not when confirming the order, so maybe change the order here to make it clearer.